### ISSUE TITLE
How to give input to two model(student, teacher) at once with one ouput?

### ISSUE THREAD

**Issue by bughunter99** [fchollet/keras#6236]
I'm converting student-teacher model in below url to keras one.

https://github.com/chengshengchan/model_compression/blob/master/teacher-student.py

How can I give input to two model(student, teacher) and get one output from only student in keras? I'll set teacher's all tensors with trainable=false, and loss function as difference between student and teacher's output like below :

`tf_loss = tf.nn.l2_loss(teacher - student)/batch_size`

As I know, it is possible to give input to only one model when defining model.fit. But in this cases, I should it to both of teacher and student model.

Thank in advance!

**Comment by bughunter99** [comment]
This had been solved like : https://stackoverflow.com/questions/43368241/student-teacher-model-in-keras
